pom.xml

```
@@ -123,6 +134,7 @@
     </dependencies>
 
     <build>
+        <finalName>${artifactId}</finalName>
```

How naming your war file this way ? :) 

```suggestion
        <finalName>warn-me</finalName>
```